src/pages/workspaces/migration/WorkspaceItem.ts

```diff
@@ -154,8 +157,9 @@ export const WorkspaceItem = (props: WorkspaceItemProps): ReactNode => {
                 tooltip: migrateStarted ? 'Migration has been scheduled' : '',
                 onClick: () => {
                   const migrateWorkspace = reportErrorAndRethrow('Error starting migration', async () => {
-                    setMigrateStarted(true);
```

Comment:
Moved the disable below the ajax call because otherwise the button stays disabled if the server call throws an error.